resources/migrations/000_migrations.yaml

```diff
@@ -14262,6 +14262,53 @@ databaseChangeLog:
         - sql: UPDATE metabase_database SET engine = 'presto-jdbc' WHERE engine = 'presto'
       rollback: # nothing to do, since we don't know which drivers used to be presto
 
+  - changeSet:
```

Comment:
this migration appears to not allow upgrading any existing actions that currently lack a database_id

![Screen Shot 2023-02-20 at 11 18 18 AM](https://user-images.githubusercontent.com/30528226/220177018-133cc303-1fef-431f-81d9-0486e7e094a8.png)